Insights/InputEntity_Account/AccountActions.yaml

```diff
@@ -0,0 +1,60 @@
+SchemaVersion: 1.0
+DataTypes:
+  - DataType: AuditLogs
+  - DataType: SecurityEvent
+Type: KQL
+Provider: Sentinel
+RequiredInputFieldsSets:
+  - - Account_Name
+    - Account_NTDomain
+  - - Account_Name
+    - Account_UPNSuffix
+  - - Account_AADUserId
+  - - Account_SID
+BaseQuery: |
+  let GetAccountActions = (v_Account_Name:string, v_Account_NTDomain:string, v_Account_UPNSuffix:string, v_Account_AADUserId:string, v_Account_SID:string){
+  AuditLogs
+  | where OperationName in~ ('Delete user', 'Change user password', 'Reset user password', 'Change password (self-service)',  'Reset password (by admin)', 'Reset password (self-service)', 'Update user')
+  | extend UserPrincipalName = tostring(TargetResources[0].userPrincipalName)
+  | extend Account_Name = tostring(split(UserPrincipalName, '@')[0])
+  | extend Account_UPNSuffix = tostring(split(UserPrincipalName, '@')[1])
+  | extend Action = tostring(parse_json(tostring(parse_json(tostring(TargetResources[0].modifiedProperties))[0])))
+  | extend ModifiedProperty = parse_json(Action).displayName
+  | extend ModifiedValue = parse_json(Action).newValue
+  | extend Account_AADUserId = tostring(TargetResources[0].id)
+  | extend DisableUser = iif(ModifiedProperty =~ 'AccountEnabled' and ModifiedValue =~ '[false]', 'True', 'False')
+  | union isfuzzy=true (
+  SecurityEvent
+  | where EventID in (4720, 4722, 4723, 4724, 4725, 4726, 4740)
+  | extend OperationName = tostring(EventID)
+  | where AccountType =~ "user" or isempty(AccountType)
+  | extend Account_Name = TargetUserName, Account_NTDomain = TargetDomainName, Account_SID = TargetSid
+  )
+  | where (Account_Name =~ v_Account_Name and (Account_UPNSuffix =~ v_Account_UPNSuffix or Account_NTDomain =~ v_Account_NTDomain)) or Account_AADUserId =~ v_Account_AADUserId or Account_SID =~ v_Account_SID
+  };
+  GetAccountActions('{{Account_Name}}', '{{Account_NTDomain}}', '{{Account_UPNSuffix}}', '{{Account_AADUserId}}', '{{Account_SID}}')
+
+Activities:
+ EnabledByDefault: true
+ Items:
+ - Id: 541ca1a7-197f-4157-8a63-9b36c446282d
+   Title: "The user has created an account"
+   Content: "The user has created account:{{Account_Name}}' '{{Count}}' times"
+   Description: "This activity displays accounts created by the user"
+   QueryDefinitions:
+    Filter: "where OperationName in~ ('Add user', '4720')"
+    SummarizeBy: "Account_Name, Account_NTDomain, Account_UPNSuffix"
+ - Id: aa8e2a70-c2f1-4e77-bb93-dfef8a553bcd
+   Title: "The user has deleted account {{Account_Name}}"
+   Content: "The user has deleted an account({{Account_Name}}) {{Count}} times"
```

Comment:
I would suggest you remove the parenthesis and use parenthesis to illustrate it could be more than once "time(s)" - 'The user has deleted the account {{Account_Name}} {{Count}} time(s)'